### PR TITLE
WinUI-3213 : Updated Customization in UG document for SfMaskedTextBox control

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
Updated customization in the masked text box UG.

### Solution Description
Added the content about customization for the masked text box control.
Added the output images for the header, header template and description.

### CODE REVIEW

File: winui/Masked-TextBox/Customization.md [hunk 1]
```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Customization in WinUI SfMaskedTextBox
\ No newline at end of file
+# Customization in WinUI MaskedTextBox
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Customization.md:10]: @Vignesh-SF3580 Learn all about the customization such as header, header template, and description support in the MaskedTextBox (SfMaskedTextBox) control.

File: winui/Masked-TextBox/Customization.md [hunk 2]
```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
...
+
+This section explains about header, header template and description properties of MaskedTextBox.
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Customization.md:13]: @Vignesh-SF3580 This section explains the header, header template, and description of MaskedTextBox.

File: winui/Masked-TextBox/Customization.md [hunk 3]
```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
...
+## Header
+
+The `Header` property is used to display the title for the MaskedTextBox Control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedTextBox x:Name="sfMaskedTextBox"
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Customization.md:21]: @Vignesh-SF3580 remove the unwanted variable name declaration and set width as 200.

File: winui/Masked-TextBox/Customization.md (hunk 1 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Customization.md:10]: Changed the description.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Masked-TextBox/Customization.md (hunk 2 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Customization.md:13]: Changed the content for customization.

File: winui/Masked-TextBox/Customization.md (hunk 3 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Customization.md]: Removed the variable name and set width as 200.

File: winui/Masked-TextBox/Customization.md [hunk 4]
```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI MaskedTextBox header](MaskedTextBox_Images/winui_masked_textbox_header.png)
+
+## Header Customization
+
+By using the `HeaderTemplate` property, you can customize the appearance of control's header. 
+
+<syncfusion:SfMaskedTextBox Width="200" MaskType="Simple" Mask=">AAAAA-AAAAA-AAAAA-AAAAA">
```
[the remark this answers is not in the document]
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Customization.md:47]: Added the xaml tags.

**Pull request merged**